### PR TITLE
Initial commit for pylatexenc

### PR DESCRIPTION
**Pull request by milljm**
My first commit. Be gentle.

I am willing to maintain this recipe. Although, I am not the owner.

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [-] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pylatexenc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/phfaist/pylatexenc/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/pylatexenc/meta.yaml]: This looks like it can be `noarch: python`

**Comment by milljm** [comment]
@scopatz I am not much of a windows guy (so please pardon my ignorance)... do I need to exclude some files? What I do know, is that the files I am _thinking_ should be whitelisted should _always_ be whitelisted (kernel.dll, etc). So why did the build system not _automatically_ whitelist them? That is... if I am even on the right path to solve this.

**Comment by scopatz** [comment]
noarch is allowed to fail on windows, because it will only be built on linux.  (It will of course be installable everywhere)

File: recipes/pylatexenc/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015-2019 Philippe Faist
```
Comment (npavlovikj) [inline recipes/pylatexenc/LICENSE.txt]: Looks like the license file is in the Github tarball, so please remove the file from here.

File: recipes/pylatexenc/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+
```
Comment (npavlovikj) [inline recipes/pylatexenc/meta.yaml]: Remove empty line.

File: recipes/pylatexenc/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  imports:
+    - pylatexenc
+
+about:
+  home: https://github.com/phfaist/pylatexenc
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simple LaTeX parser providing latex-to-unicode and unicode-to-latex conversion'
+
```
Comment (npavlovikj) [inline recipes/pylatexenc/meta.yaml]: You can add:
```
doc_url: https://pylatexenc.readthedocs.io/en/latest
dev_url: https://github.com/phfaist/pylatexenc
```
in the `about` section.

File: recipes/pylatexenc/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/phfaist/pylatexenc/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
```
Comment (npavlovikj) [inline recipes/pylatexenc/meta.yaml]: I see there are entry points in setup.py:
```
    entry_points = {
        'console_scripts': [
            'latexwalker=pylatexenc.latexwalker.__main__:main',
            'latex2text=pylatexenc.latex2text.__main__:main',
            'latexencode=pylatexenc.latexencode.__main__:main',
```
Do you know if this package can be used from the command line, e.g. `latexwalker  -h`?

If so, that please add the entry points here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points, and test for them as commands in the test section.

Comment (milljm) [reply recipes/pylatexenc/meta.yaml]: Looks like it does. Will do.

**Comment by mwcraig** [comment]
Merging (this is `noarch` so the windows fails are ok), thanks @milljm!

**Comment by mwcraig** [comment]
Closing manually; I merged while github was having an issue with its api. This has been merged -- see bd1ec70704122c7393c01d1904ff89e49b1680d4

**Pull request closed without merging**